src/docker-compose.override.yml

```diff
@@ -5,7 +5,7 @@ services:
     environment:
       - ASPNETCORE_ENVIRONMENT=Development
       - ARCUS_HTTP_PORT=88
-      - ARCUS_EVENTGRID_TOPICENDPOINT=
-      - ARCUS_EVENTGRID_AUTHKEY=
+      - ARCUS_EVENTGRID_TOPICENDPOINT=https://arcus-event-grid-dev-we-integration-tests.westeurope-1.eventgrid.azure.net/api/events
+      - ARCUS_EVENTGRID_AUTHKEY=B7rqeccGOOhhI0arirZNx5fyM/yJ2RZizPsybLZXkMI=
```

Comment:
Woops, that's not something we want in source control. Maybe regenerate ir or so to be safe 😢 .

Reply:
Reverted and gitignored, will squash PR anyway